src/scripts/app.js

```diff
@@ -37,12 +49,13 @@ class App extends JSXComponent {
         return response.json();
       })
       .then((response) => {
-        if (response.statusCode >= 400) {
+        
+        // Response doesn't contain status code in here.
+        if (!response.hasOwnProperty("html") ) {
```

Comment:
While debugging that on Firefox by dumping `response` to the console, the `statusCode` property was not present. If it was on the prototype then it is my bad because those things doesn't appear on the log but at least here, at first glance the code appeared broken so I changed it for some property known to be on the object and not on the prototype chain... does this make sense?